bitnami/prometheus-operator/README.md

```diff
@@ -351,6 +351,10 @@ This chart includes a `values-production.yaml` file where you can find some para
 +   logLevel: error
 ```
 
+### <a name="additionalScrape"></a>Additonal scrape configurations
```

Comment:
fix typo and remove the anchor tag. Markdown automatically creates (#additional-scrape-configurations)

```suggestion
### Additional scrape configurations
```